openkfw/TruBudget
# 1851 add create project button in card view

### Checklist




### Description
- added create project button in card view, so that user does not have to do unnecessary operations like scroll and then click on empty card or select table view and then click on add new project button

<img width="1154" alt="Screenshot 2024-05-29 at 09 26 41" src="https://github.com/openkfw/TruBudget/assets/55734106/47a91593-6cfe-489e-8aba-aa4f95c705a0">


### How to test

1. login as mstein
2. select card view of projects
3. see that on the right side of top pagination there is a plus button which can add a new project now


Closes #1851
